boards/arm/mr_canhubk3/mr_canhubk3.dts

```diff
@@ -409,10 +409,18 @@
 	phy-connection-type = "rmii";
 	local-mac-address = [02 04 9f aa bb cc];
 	status = "okay";
+};
 
-	fixed-link {
-		speed = <100>;
-		full-duplex;
+&mdio0 {
+	pinctrl-0 = <&mdio0_default>;
+	pinctrl-names = "default";
+	status = "okay";
+
+	phy: ethernet-phy@18 {
```

Comment:
unit address is in hex, it should be `@12`, and for consistency perhaps also in `reg = <0x12>`.

Reply:
Changed and tested.